src/main/java/Page.java

```diff
@@ -0,0 +1,222 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+
+public class Page {
+
+    private Scanner scan;
```

Comment:
Would it be better if we use `scanner` for the variable name? I've noticed similar issues in many other places as well.